src/main/java/org/jenkinsci/plugins/workflow/cps/DSL.java

```diff
@@ -132,9 +132,9 @@ public Object invokeMethod(String name, Object args) {
         try {
             for (Class<?> e : Index.list(Symbol.class, Jenkins.getActiveInstance().pluginManager.uberClassLoader, Class.class)) {
                 if (Descriptor.class.isAssignableFrom(e)) {
-                    Symbol s = e.getAnnotation(Symbol.class);
+                    String s = SymbolLookup.getSymbolValue(e);
                     if (s != null) {
-                        symbols.addAll(Arrays.asList(s.value()));
+                        symbols.addAll(Arrays.asList(s));
```

Comment:
(Which is something you need to think about in the other downstream PRs. What is your behavior when there are multiple allowed symbols? Probably rather than `@CheckForNull String` you need to return `@Nonnull Set<String>`.)